### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Tapanito**
Addresses #916

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -285,6 +285,18 @@ If you want to compile the applications under ``examples/``, you need
 to remove or rename the ``event.h`` from libev's installation, because
 it conflicts with libevent's installation.
 
+Notes for installation on Linux systems
+--------------------------------------------
+After installing nghttp2 tool suite with `make install` one might experience a similar error:
+
+```
+nghttpx: error while loading shared libraries: libnghttp2.so.14: cannot open shared object file: No such file or directory
+```
```
Comment (tatsuhiro-t) [inline README.rst]: This reST formatted document, and the verbatim text is achieved using ".. code-block:: text" like this:

```
.. code-block:: text

    nghttpx: error while loading shared libraries: libnghttp2.so.14: cannot open shared object file: No such file or directory
```

File: README.rst [hunk 2]
```diff
@@ -285,6 +285,18 @@ If you want to compile the applications under ``examples/``, you need
 to remove or rename the ``event.h`` from libev's installation, because
 it conflicts with libevent's installation.
 
+Notes for installation on Linux systems
+--------------------------------------------
+After installing nghttp2 tool suite with `make install` one might experience a similar error:
```
Comment (tatsuhiro-t) [inline README.rst]: This is reST document, and monospace text should be enclosed with double backtick like this:

```
``make install``
```

File: README.rst [hunk 3]
```diff
@@ -287,11 +287,12 @@ it conflicts with libevent's installation.
 
 Notes for installation on Linux systems
 --------------------------------------------
-After installing nghttp2 tool suite with `make install` one might experience a similar error:
+After installing nghttp2 tool suite with ``make install`` one might experience a similar error:
+
+.. code-block:: text
 
-```
 nghttpx: error while loading shared libraries: libnghttp2.so.14: cannot open shared object file: No such file or directory
```
Comment (tatsuhiro-t) [inline README.rst]: I might not be clear enough, this line should be indented 4 spaces.

Comment (Tapanito) [inline README.rst]: Sorry, my bad!

**Pull request merged**